README.md

```diff
@@ -26,6 +26,11 @@ gpg:                using RSA key 44CD70EA353C06DE5AB608E4D842AF7166B99EDA
 gpg:                issuer "integrations@actbluetech.com"
 gpg: Good signature from "integrations@actbluetech.com" [full]
 ```
+4. Create a distributable zip archive of the plugin for upload as a Github release. The script does require that `GPG_PRIVATE_KEY` and `GPG_UID` are set as environment variables on your computer.
```

Comment:
This step should be in the Installation section further below in this file.

Reply:
I don't believe it should fall there. Those instructions depend on running CirlceCI and it is a step in CircleCI that performs the manual steps outlined here.

Reply:
It should definitely be in the "Releasing a new version" section and it should definitely not be in the "Verifying GitHub Releases" section.

The "Verifying GitHub Releases" section documents the steps _a WordPress administrator_ can take to verify the signature of a released version of the plugin to ensure that the package has not been modified to effect a supply chain attack.

The "Releasing a new version" section documents the steps _an ATS developer_ takes in order to create and new distributable version of the plugin and tag in Github.

The content in question here, which documents signing a new distributable version of the plugin (i.e., the zip file), is, by definition, a release step and is _action taken by an ATS developer_, not a WordPress administrator.

Reply:
My bad, I thought it was there as that was where I asked for it to be placed. @julissa it does seem to be in the wrong section.